tests/unit/lib/test_datachain.py

```diff
@@ -770,6 +770,26 @@ def test_collect_nested_feature(test_session):
         assert nested == features_nested[n]
 
 
+def test_select_no_sys_id(test_session):
```

Comment:
not clear from the name what we are testing, can we rephrase it?

Reply:
Done